sdk/resourcemanager/Azure.ResourceManager/README.md

```diff
@@ -52,33 +52,51 @@ Additional documentation for the `Azure.Identity.DefaultAzureCredential` class c
 
 To reduce both the number of clients needed to perform common tasks and the amount of redundant parameters that each of those clients take, we have introduced an object hierarchy in the SDK that mimics the object hierarchy in Azure. Each resource client in the SDK has methods to access the resource clients of its children that is already scoped to the proper subscription and resource group.
 
-To accomplish this, we're introducing 4 standard types for all resources in Azure:
+To accomplish this, we're introducing 3 standard types for all resources in Azure:
 
-#### [Resource]Data
-This represents the data that makes up a given resource. Typically, this is the response data from a service call such as HTTP GET and provides details about the underlying resource. Previously, this was represented by a **Model** class.
+#### **[Resource].cs**
 
-#### [Resource]Operations
+This represents a full resource client object which contains a **Data** property exposing the details as a **[Resource]Data** type.
+It also has access to all of the operations on that resource without needing to pass in scope parameters such as subscription ID or resource name.  This makes it very convenient to directly execute operations on the result of list calls
+since everything is returned as a full resource client now.
 
-This represents a service client that's scoped to a particular resource. You can directly execute all operations on that client without needing to pass in scope parameters such as subscription ID or resource name.
+```csharp
+ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+string rgName = "myResourceGroup";
+ResourceGroup rg = await armClient.DefaultSubscription.GetResourceGroups().GetAsync(rgName);
+await foreach (VirtualMachine vm in rg.GetVirtualMachines().GetAllAsync())
+{
+    //previously we would have to take the resourceGroupName and the vmName from the vm object
+    //and pass those into the powerOff method as well as we would need to execute that on a separate compute client
+    await vm.StartPowerOff().WaitForCompletionAsync();
+}
+```
 
-#### [Resource]Container
+#### **[Resource]Data.cs**
+
+This represents the data that makes up a given resource. Typically, this is the response data from a service call such as HTTP GET and provides details about the underlying resource. Previously, this was represented by a **Model** class.
+
+#### **[Resource]Container.cs**
 
 This represents the operations you can perform on a collection of resources belonging to a specific parent resource.
-This mainly consists of List or Create operations. For most things, the parent will be a **ResourceGroup**. However, each parent / child relationship is represented this way. For example, a **Subnet** is a child of a **VirtualNetwork** and a **ResourceGroup** is a child of a **Subscription**.
+Most of the logical collection operations can be mapped to a function operation on this object
```

Comment:
"mapped to a function operation" reads a little like you're trying to sell me on programming in Haskell.  Could we just say "This object provides most of the logical collection operations."?